packages/mdc-drawer/README.md

```diff
@@ -68,6 +68,8 @@ However, you can also use SVG, [Font Awesome](https://fontawesome.com/), or any
 ```scss
 @import "@material/drawer/mdc-drawer";
 @import "@material/list/mdc-list";
+// For dismissible drawer, include mdc-side-sheet for the shared mdc-side-sheet-app-content class
```

Comment:
Would this be a good candidate for the `mdc-base-emit-once()` mixin?

Reply:
There are probably several places we could make this argument... so far we've only used emit-once for the packages that support text field / select and are basically never used standalone (line-ripple, notched-outline, floating-label).

It occurs to me that future discussions on how we organize our styles might impact how we use emit-once, so I'm going to hold off on opting into this here for now, but by the time we return to side sheet to finish it, we might have a better idea.